mrp_load/workcenter_view.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+
+<!--Workcenters-->
+
+<record id="mrp_workcenter_tree_view" model="ir.ui.view">
+    <field name="model">mrp.workcenter</field>
+    <field name="inherit_id" ref="mrp.mrp_workcenter_tree_view"/>
+    <field name="arch" type="xml">
+        <field name="resource_type" position="before">
+            <field name="load" widget="float_time" sum="Load"/>
+            <field name="availability"/>
+            <button name="toogle_online"
+                    string="Click to set online => will be used in capacity computation"
+                    icon="gtk-disconnect"
+                    attrs="{'invisible': [('online', '=', True)]}"
+                    type="object"/>
+            <button name="toogle_online"
+                    string="Click to set offline => workcenters children will also set offline"
+                    icon="gtk-connect"
+                    attrs="{'invisible': [('online', '=', False)]}"
+                    type="object"/>
+            <field name="online" invisible="1"/>
+        </field>
+        <xpath expr="//tree[@ string='Work Center']" position="attributes">
+            <attribute name="colors">red:load>0 and online==False;grey:online==False</attribute>
+        </xpath>
+    </field>
+</record>
+
```

Comment:
Maybe you should add a button next to workcenter in tree view for notice people they can change it easily.